app/components/Countdown.tsx

```diff
@@ -0,0 +1,88 @@
+import * as moment from "moment";
```

Comment:
please make this component only about displaying the countdown. No additional markup, like: `Reward starting point` This component should be reusble and it's not becuase of that coupling

Reply:
good point